README.md

```diff
@@ -55,13 +55,63 @@ The server can currently run in two modes: `standalone` and `forwarder`.  It is
+- `max-workers`: the number of aggregators to process metrics.  Defaults to the number of logical cores.
+- `max-queue-size`: the size of the buffers between parsers and workers.  Defaults to `10000`, monitored via
+  `channel.*` metric, with `dispatch_aggregator_batch` and `dispatch_aggregator_map` channels.
+- `max-concurrent-events`: the maximum number of concurrent events to be dispatching.  Defaults to `1024`, monitored
+  via `channel.*` metric, with `backend_events_sem` channel.
+- `estimated-tags`: provides a hint to the system as to how many tags are expected to be seen on any particular metric,
+  so that memory can be pre-allocated and reducing churn.  Defaults to `4`.  Note: this is only a hint, and it is safe
+  to send more.
+- `log-raw-metric`: logs raw metrics received from the network.  Defaults to `false`.
+- `metrics-addr`: the address to listen to metrics on. Defaults to `:8125`.
+- `namespace`: a namespace to prefix all metrics with.  Defaults to ''.
+- `statser-type`: configures where internal metrics are sent to.  May be `internal` which sends them to the internal
+  processing pipeline, `logging` which logs them, `null` which drops them.  Defaults to `internal`, or `null` if the
+  NewRelic backend is enabled.
+- `percent-threshold`: configures the "percentiles" sent on timers.  Space separated string.  Defaults to `90`.
+- `heartbeat-enabled`: emits a metric named `heartbeat` every flush interval, tagged by `version` and `commit`.
+  Defaults to `false`.
+- `receive-batch-size`: the number of datagrams to attempt to read.  It is more CPU efficient to read multiple, however
+  it takes extra memory.  See [Memory allocation for read buffers] section below for details.  Defaults to 50.
+- `conn-per-reader`: attempts to create a connection for every UDP receiver.  Not supported by all OS versions.
+  Defaults to `false`.
+- `bad-lines-per-minute`: the number of metrics which fail to parse to log per minute.  This is used to prevent a bad
+  client spamming malformed statsd data, while still logging some information to enable troubleshooting.  Defaults to `0`.
+- `hostname`: sets the hostname on internal metrics
+- `timer-histogram-limit`: specifies the maximum number of buckets on histograms.  See [Timer histograms] below.
+
+
 In `forwarder` mode, raw metrics are collected from a frontend, and instead of being aggregated they are sent via http
 to another gostatsd server after passing through the processing pipeline (cloud provider, static tags, filtering, etc).
 
 A `forwarder` server is intended to run on-host and collect metrics, forwarding them on to a central aggregation
 service.  At present the central aggregation service can only scale vertically, but horizontal scaling through
 clustering is planned.
 
+Aligned flushing is deliberately not supported in `forwarder` mode, as it would impact the central aggregation server
```

Comment:
Also, distributed sync is hard? :badpokerface:

Reply:
For forwarder it's more about "if 1000 hosts all send at exactly 8:34:20pm, then the central server blows up" vs "if 100 send at 8:34:30, 100 send at 8:34:31, 100 send at 8:34:32, etc...".  It's not even that bad because it'll be skewed out across milliseconds, not seconds.

But even then that's not the actual point.  The point is that if you have two independent services in us-east-1 and us-west-2, and they're out of phase by 5 seconds, it makes the backend a bit wonky when you query it.

So it's really about the **timestamp** on the data, not the time the data is sent.